Properties/Resources.resx

```diff
@@ -1119,156 +1092,6 @@
   <data name="Down" xml:space="preserve">
     <value>Down</value>
   </data>
-  <data name="Downloads_AnimeBytes" xml:space="preserve">
-    <value>AnimeBytes Settings</value>
-  </data>
-  <data name="Downloads_AnimeBytesDetails" xml:space="preserve">
-    <value>Please enter Anime Bytes login details</value>
-  </data>
-  <data name="Downloads_AnimeBytesInfo" xml:space="preserve">
-    <value>AnimeBytes is an anime torrent site which specializes in full series downloads, however it requires an account first</value>
-  </data>
-  <data name="Downloads_BakaBT" xml:space="preserve">
-    <value>BakaBT Settings</value>
-  </data>
-  <data name="Downloads_BakaBTDetails" xml:space="preserve">
-    <value>Please enter BakaBT login details</value>
-  </data>
-  <data name="Downloads_BakaBTInfo" xml:space="preserve">
-    <value>BakaBT is an anime torrent site which specializes in full series downloads, however it requires an account first</value>
-  </data>
-  <data name="Downloads_Connected" xml:space="preserve">
-    <value>Connected sucessfully</value>
-  </data>
-  <data name="Downloads_Details" xml:space="preserve">
-    <value>Details</value>
-  </data>
-  <data name="Downloads_DisableRefresh" xml:space="preserve">
-    <value>Only use when auto refresh is disabled</value>
-  </data>
-  <data name="Downloads_Download" xml:space="preserve">
-    <value>Download</value>
-  </data>
-  <data name="Downloads_Failed" xml:space="preserve">
-    <value>Failed! See log for more details if needed</value>
-  </data>
-  <data name="Downloads_GoTo" xml:space="preserve">
-    <value>Go to Website</value>
-  </data>
-  <data name="Downloads_GoWebsite" xml:space="preserve">
-    <value>Go to Website</value>
-  </data>
-  <data name="Downloads_Results" xml:space="preserve">
-    <value>{0} Results</value>
-  </data>
-  <data name="Downloads_Searching" xml:space="preserve">
-    <value>Searching...</value>
-  </data>
-  <data name="Downloads_TorrentBlackHole" xml:space="preserve">
-    <value>Torrent Black Hole Settings</value>
-  </data>
-  <data name="Downloads_TorrentDefault" xml:space="preserve">
-    <value>Default Torrent Sources for Searching</value>
-  </data>
-  <data name="Downloads_uTorrentDetails" xml:space="preserve">
-    <value>Please enter all uTorrent details</value>
-  </data>
-  <data name="Downloads_uTorrentWebUI" xml:space="preserve">
-    <value>uTorrent WebUI details</value>
-  </data>
-  <data name="Download_AnimeBytesOnly" xml:space="preserve">
-    <value>AnimeBytes ONLY for Series</value>
-  </data>
-  <data name="Download_Approval" xml:space="preserve">
-    <value>Approval</value>
-  </data>
-  <data name="Download_AutoRefresh" xml:space="preserve">
-    <value>Auto Refresh</value>
-  </data>
-  <data name="Download_BakaBTOnly" xml:space="preserve">
-    <value>BakaBT ONLY for Series</value>
-  </data>
-  <data name="Download_Bookmark" xml:space="preserve">
```

Comment:
This is one is needed

Reply:
BakaBT functionality should of been removed.

Reply:
I think I hit the wrong lines for AB and BBT

Reply:
Bookmark and approval are needed